module-2/backend/week_two.md

```diff
@@ -25,30 +32,70 @@ end
 ```
 
 Now how would you find the owner of the team with an id of 4?
-
+<p>
+Answered above. Another way would be to set the team with an id of 4 to a variable and call owner on that, 'team = Team.find(4)' 'team.owner'
```

Comment:
👍 

Above I think we were assuming that there were no changes to the model, which would require some less than ideal code like the following:

```ruby
owner_4_id = Team.find(4).owner_id
owner = Owner.find(owner_4_id)
```